tests/reprsimil/test_brsa.py

```diff
@@ -110,51 +117,75 @@ def test_fit():
 
 
     # Test fitting with GP prior.
-    brsa = BRSA(GP_space=True,GP_inten=True,verbose=False,n_iter = 200)
+    brsa = BRSA(GP_space=True,GP_inten=True,verbose=False,n_iter = 200,auto_nuiance=False)
 
-    brsa.fit(X=Y, design=design.design_used, scan_onsets=scan_onsets,
+    # We also test that it can detect baseline regressor included in the design matrix for task conditions
+    wrong_design = np.insert(design.design_task, 0, 1, axis=1)
+    with pytest.raises(ValueError) as excinfo:
+        brsa.fit(X=Y, design=wrong_design, scan_onsets=scan_onsets,
+             coords=coords, inten=inten)
+    assert 'Your design matrix appears to have included baseline time series.' in str(excinfo.value)
```

Comment:
The `pytest.raises` above will fail the test if a `ValueError` exception is **not** raised. Therefore, there is no need for this assertion, unless you are concerned a different `ValueError` is raised, unrelated to the design matrix including baseline time series.